satellitepy/data/tools.py

```diff
@@ -148,6 +153,15 @@ def save_patches(
         label_name = label_path.name if label_path is not None else None
         logger.info(f"{img_path.name}, {label_name}, {mask_name}")
 
+        # Change output directory to patches if subset_data is not None
+        if subset_data is not None:
+            subset = subset_data.loc[subset_data['image'] == '/mnt/2tb-1/satellitepy/data/FR24_dataset/images/' + img_path.name]
```

Comment:
Please remove the hardcoded path as this will break if it should be changed or somebody else wants to use this script